### ISSUE TITLE
Question: Can we disable PRE_COMMIT_COLOR from .pre-commit.config.yaml

### ISSUE THREAD

**Issue by razy69** [pre-commit/pre-commit#2696]
Hi,

We are facing an issue using pre-commit -> tox -> poetry.
I opened a case at Tox repository: https://github.com/tox-dev/tox/discussions/2857

The only way to get it works together is to disable PRE_COMMIT_COLOR.
So i have to exec:

```console
PRE_COMMIT_COLOR=never pre-commit run tox-code-checks
```

I would like to not set it through env var, could i set it in the pre-commit yaml config ?

Thanks.

**Comment by asottile** [comment]
nope, it can only be set through the command line or an environment variable

**Comment by asottile** [comment]
the full configuration is documented on the website: https://pre-commit.com

**Comment by razy69** [comment]
Ok this is what i through reading the documentation, but could we consider to add this feature ?

**Comment by asottile** [comment]
no. your use case is a bad one anyway and there are already two ways to accomplish what you want

**Comment by razy69** [comment]
Sorry to bother you, but in what this workflow isn't good ?
Could you give me advices ?

Thanks anyway.

**Comment by asottile** [comment]
using pre-commit to run tox to run poetry to run X is insane -- you'd do much better to just have pre-commit run X -- or at the very least not tox or not poetry

kind of the point of pre-commit is it manages your dependencies -- putting tox or poetry in the mix is a bad idea

**Comment by razy69** [comment]
The main idea behind is to specify dependencies at one place (pyproject.toml).

To me pre-commit is use-full to setup git hooks from a yaml, not managing dependencies (i am probably wrong..).
Tox isolate tests envs by creating venv for each target.

We have 5 targets:

 - generate documentation from code docstring
 - tests our code using pytest (we have also testing helpers packages)
 - format the code (black and isort)
 - quality checks (ruff and mypy)
 - security checks (bandit and safety)

So i will have venvs created for each target, and inside of them have only the set of dependencies needed (thanks to poetry dependency groups)

I just have to execute `tox -p` to run all tests.
Then pre-commit only deserves to me for checking commit message and running tox.

So according to your reply, i have to duplicate dependency specifications, or maybe generate an old-fashioned requirement.txt from poetry, to use it in tox ?

Also i don't want to duplicate checks declaration, tox/pre-commit..
